src/remotestorage.js

```diff
@@ -538,14 +538,32 @@
     /**
      * Method: setApiKeys (experimental)
      *
-     * Set API keys for (currently) GoogleDrive and/or Dropbox backend support.
-     * See also the 'backends' example in the starter-kit. Note that support for
-     * both these backends is still experimental.
+     * Set API keys for (currently) GoogleDrive, Dropbox and SAFE Network backend 
+     * support.
+     * 
+     * mrhTODO the following is changing needs updating once new scheme clear
+     * 
+     * Note that SafeNetwork (SAFE Network) backend does not require an API key from
+     * an authority as in the case of most cloud services, so setApiKeys() is
+     * used to define the application related information that is passed to 
+     * an authorisation gateway (SAFE Launcher). The gateway can return a stored
+     * authorisation token or may asks the user for permission to generating one.
+     * 
+     * See also the 'backends' example in the starter-kit (deprecated). Note that 
+     * support for both these backends is still experimental.
```

Comment:
This function is used for setting API keys, as the name suggests. It doesn't make sense to use it for anything else. Either add a new function that expresses what is set for SN, or add more arguments to `connect`.